provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerController.java

```diff
@@ -35,48 +36,26 @@
   produces = {"application/json", "application/fhir+json", "application/json+fhir"}
 )
 public class PractitionerController {
+
   private Transformer transformer;
 
   private Bundler bundler;
 
-  private PpmsClient client;
+  private PpmsClient ppmsClient;
 
   /** Controller setup. */
   public PractitionerController(
       @Autowired Transformer transformer,
       @Autowired Bundler bundler,
-      @Autowired PpmsClient client) {
+      @Autowired PpmsClient ppmsClient) {
```

Comment:
As a little more clean-up, Lombok can generate this constructor for us: `@AllArgsConstructor(onConstructor = @__({@Autowired}))`